motr/io_req_fop.c

```diff
@@ -101,6 +101,143 @@ M0_INTERNAL struct m0_file *m0_client_fop_to_file(struct m0_fop *fop)
 	return &ioo->ioo_flock;
 }
 
+/**
+ * Copies correct part of attribute buffer to client's attribute bufvec.
```

Comment:
This comment has become redundant due to the subsequent comment, this comment can be removed.